selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/ActivityStream.java

```diff
@@ -99,6 +99,10 @@ public void checkActivity(String name) {
     $(byText(name)).isDisplayed();
     info("The activity of the name:" + name + " is shown successfully");
   }
+  public void checkTextActivity (String desc){
+    ELEMENT_TEXT_ACTIVITY.shouldHave(Condition.text("desc"));
```

Comment:
1. you can use checkActivity() (method already exist in ActivityStream.java), should not duplicate code.
2. in this methode you check the String "desc", not the variable desc.